tests/dao_test.js

```diff
@@ -0,0 +1,1294 @@
+'use strict';
+
+const { expect } = require('chai'),
+  { Sequelize, DataTypes } = require('../source'),
+  Op = Sequelize.Op;
+
+// Skip reason: This test implements both HSTORE and an array of JSONB.
+// CRDB does not support these.
+describe.skip('[POSTGRES Specific] DAO', () => {
```

Comment:
Can we simply leave the body of this blank since we're skipping all of this anyway? I feel that it is fairly unnecessary to check in the extra code but leaving the describe.skip here is useful for the reason

Reply:
Will do.